spec/features/record_show_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+describe GenericFile do
+  context 'record', :type => :feature do
+
+    let(:user) { FactoryGirl.find_or_create :user_with_fixtures }
+    let!(:file) do
+      GenericFile.new.tap do |f|
+        f.title = ['little_file.txt']
+        f.creator = ['little_file.txt_creator']
+        f.resource_type = ["Thesis" ]
+        f.read_groups = ['public']
+        f.abstract = "http://hydranorthdev.library.ualberta.ca/advanced"
```

Comment:
I think the problem #704 is meant to solve is for collection and files' abstract and descriptions where there is content like
```
This item is a resource in the University of Alberta Libraries' Dataverse Network. Access this item in Dataverse by 
clicking on the DOI link. | Kind of data: Audio recordings (.wav) | <p><b>Contents:</b></p> <p><ul><li>Tr. 1 – 
“Skagit for Young People,” Book 1, Tape 3, Lesson 20 (5:46)</li> <li>Contents:Tr. 1 – “Skagit for Young People” 
#16 (4:10); Comparative Vocabulary – Lushootseed (2:35)</li></ul></p> <br/> <p>From the Thom Hess 
Collection, Ethnomusicology Archives (UWEA 95-19.52; 95-19.53) University of Washington, Seattle</p> 
```